apps/admin_panel/assets/src/omg-create-account-modal/index.js

```diff
@@ -65,7 +65,7 @@ class CreateAccountModal extends Component {
       this.setState({ stage: 'finished' })
     } else {
       this.setState({
-        error: result.data.data.code,
+        error: result.data.data.description,
```

Comment:
I think it would make sense to display both the error code and the description. Error codes are always `something_happened`, so you could beautify it into:

`Something happened`
`This is the description I got from the server`.

Or, another approach is to show the description first, like how you would see in some OS:

`This is the description I got from the server`
`Error code: "something_happened"` 

So users can just give us the error code :)

Reply:
This can be changed everywhere in another PR btw!

Reply:
ideally, `description` should not be use in anycase since it probably need to be localized, but yeah showing error code is good too will change that

Reply:
I agree with that, but we're not localizing yet. If you wanna go that way, duplicate the errors descriptions in the app the way the mobile apps did it and use the error code to find the right local description.